project/build.sbt

```diff
@@ -18,6 +18,7 @@ val `bloop-build` = project
     addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.2"),
     addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M13-2"),
     addSbtPlugin("org.scalameta" % "sbt-mdoc" % "1.2.10"),
+    addSbtPlugin("org.scala-debugger" % "sbt-jdi-tools" % "1.1.1"),
```

Comment:
🤔 Why is this required?

Reply:
All of our running/testing in projects that require debugging/JDK APIs is forked, which means we system process to the installed JDK. If I read the motivation of the plugin correctly, the plugin is not useful when doing forked execution, so I'd be inclined to remove it. Did you get errors without it?

Reply:
Sorry, should have been more clear:

It adds `tools.jar` to our development classpath and makes sure it is not published with bloop as it would be illegal.
We need this jar on classpath to work on `DebugExtensions` which heavily rely on the `com.sun.jdi` package. And we need `DebugExtensions` to even instantiate the Debug Adapter Server.

Without it I get this compilations error:
```
[error] bloop/frontend/src/main/scala/bloop/dap/DebugExtensions.scala:11:16: object jdi is not a member of package com.sun
[error] import com.sun.jdi.{ObjectReference, StackFrame, ThreadReference, Value}
```

Reply:
Right, so I guess we get this error because sbt is blacklisting the load of `com.sun.jdi` when compiling these projects. If those errors com from inside the sbt shell, that makes sense.